test/EVMHost.h

```diff
@@ -127,6 +127,8 @@ class EVMHost: public evmc::MockedHost
 	static evmc::Result resultWithGas(int64_t gas_limit, int64_t gas_required, bytes const& _data) noexcept;
 	static evmc::Result resultWithFailure() noexcept;
 
+	static const std::array<evmc::bytes32, 2> m_blob_hashes;
```

Comment:
Indeed, I changed it to what you suggested. My initial thought was that it could be used in the `EVMInstructionInterpreter` as well, but it turned out that it doesn't.